examples/hello-world/main.rs

```diff
@@ -39,11 +43,11 @@ impl From<RequestError> for HelloError {
 }
 
 async fn serve(port: u16) -> io::Result<()> {
-    let mut app = App::<_, HelloError, 0, 1>::with_state(RwLock::new("Hello".to_string()));
+    let mut app = App::<_, HelloError, Ver01>::with_state(RwLock::new("Hello".to_string()));
```

Comment:
I'm not really familiar with the code in this repo, but it looks like there are multiple different names for the same thing. Is this intentional?

There's `Ver01` here.

Reply:
It's not so much intentional as the inevitable consequence of how rust's doctest works. Each doctest instance is basically its own isolated sandbox. Likewise for examples. There's nothing special about these aliases, they're just convent for illustrating that, in real usage, you would be using something external rather than hardcoding `StaticVersion<0, 1>` as a parameter.